docs/contributing.md

```diff
@@ -99,12 +102,141 @@ are running as expected.
 
 ## Style Guides
 
+In an effort to maintain consistency, Python-Markdown follows the following
```

Comment:
I am not a native speaker, but “follows the following” does not sound nice to me. Maybe “adheres to the following”?

Reply:
Good catch. I didn't notice that after multiple readings.